docs/UserGuide.md

```diff
@@ -80,8 +80,79 @@ hardware, os configuratoin, etc (unless available through the api) out of scope.
 
 # Example use cases
 
+To Compare a known valid reference configuration with a live cluster:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry>`
+
+To Compare a known valid reference configuration with a local set of CRs:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f <inputConfiguration>`
+
+To Compare a known valid reference configuration with a live cluster and with a user config:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -c <userConfig>`
+
+To Run a known valid reference configuration with a must-gather output:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f "must-gather*/*/cluster-scoped-resources","must-gather*/*/namespaces" -R`
+
 # Understanding the output
 
 # Options and advanced usage
 
+## Kubectl Environment Variables
+The tool is responsive to KUBECTL_EXTERNAL_DIFF environment variable (same as oc diff). This allows you to tailor the output formatting to suit your preference. 
+
+side-by-side comparison (total width 150 characters) with:
+`KUBECTL_EXTERNAL_DIFF="diff -y -W 150"`
```

Comment:
I think we coud explain what each option individually achieves.
-y : side by side comparison
--color: colored output
-W <char_wifth>: width of char_length
And then say. For more available options do `diff --help`